lib/cli.coffee

```diff
@@ -25,12 +26,20 @@ installMycha = (testHelperPath, done) ->
   installer.install done
 
 
+loadConfig = (done) ->
```

Comment:
I personally like exporting a function for files that are nouns ("config_loader" in this case).
Otherwise I feel the file needs to be renamed to "load_config".